primeqa/boolqa/score_normalizer/score_normalizer.py

```diff
@@ -87,3 +90,64 @@ def create_features(self, qtc_is_boolean_label, qa_pred):
         feature_list = [question_label,b_score,e_score,na_score]
         features = numpy.array(feature_list).reshape(1, -1)
         return features
+
+    def train(self,input_file : str, 
+            gold_file : str,
+            output_dir : str,
+            qtc_is_boolean_label : str = 'boolean',
+            evc_no_answer_class : str = 'no_answer'):
+        
+        numpy.random.seed(42)
+        
+        qa_pred_data = create_dataset_from_run_mrc_output(input_file, unpack=True)
+        dataset = datasets.load_dataset('json', data_files={'validation': gold_file})['validation']
+        
+        #YN_Gold = [] # is it is a YN question
+        HSA_Gold = [] # if it has a SA 
+        QSA_Scores = [] # the SA score for the QA SA prediction/ not currently used/ might want to experiment
+        YN_Pred = [] #if it was predicted to be YN
+        #EVC_Scores = [] # not used in this version
+        B_Scores = []
+        E_Scores = []
+        NA_Scores = []
+        
+        gold_annotations = dataset['annotations']
+        for i, qa_pred in enumerate(qa_pred_data):
```

Comment:
I think the format in this file is still for downloadable HF TyDi instead of Google TyDi as in leaderboard branch. Not sure what is the right thing to do here.

Reply:
This is for training - I think it should be ok here.  Maybe later we will need to make it configurable.